Dockerfile

```diff
@@ -29,3 +29,7 @@ ENV PATH $PATH:/go/bin
 ENV GOPATH=/go
 
 RUN go get github.com/fabric8io/gobump
+
+# installing latest version of git
+RUN yum remove -y git
```

Comment:
it's probably cleaner to just install the correct version on line 3 of this dockerfile, rather than installing then uninstalling and installing a newer version